pdns/recursordist/rec-main.cc

```diff
@@ -1303,7 +1337,7 @@ void broadcastFunction(const pipefunc_t& func)
   }
 
   unsigned int thread = 0;
-  for (const auto& threadInfo : RecThreadInfo::infos()) {
+  for (auto& threadInfo : RecThreadInfo::infos()) {
```

Comment:
If we make `getPipes()` `const`:
```suggestion
  for (const auto& threadInfo : RecThreadInfo::infos()) {
```